### PR TITLE
Link is broken— maybe fixed it?

### PR DESCRIPTION
**Pull request by georgia-at-pieces**

### CODE REVIEW

**Comment by georgia-at-pieces** [comment]
The edits to the plugin pages are not related to the Pieces OS link (I think it might be bad PR practice to have them all on the same one? But I don't know how to fix it)

**Comment by jordan-pieces** [comment]
all good it should be totally fine, ill review this

File: docs/extensions-plugins/jetbrains.mdx [hunk 1]
```diff
@@ -155,14 +155,14 @@ To open and edit a previously saved snippet:
 You can adjust your snippet's name, description, and classification from the right-click context menu.
 
 ## Use Suggested Save in JetBrains
-Pieces can intelligently identify and save snippets based on re-use patterns in your JetBrains workflow. Whenever Pieces saves a snippet, you'll see a "Saved a Piece!" confirmation in the bottom right corner of your screen. Your snippet is now saved in Pieces with a default name and automatically classified by language.
+Pieces can intelligently identify and save snippets based on re-use patterns in your JetBrains workflow. Whenever Pieces saves a snippet, you'll see a "Saved a Piece!" confirmation in the bottom right corner of your screen. Your snippet is now saved in Pieces with auto-generated metadata. 
 
 <Image zoom src = "https://storage.googleapis.com/pieces_static_resources/jetbrains_marketplace/gifs/SUGGESTED_SNIPPETS_JETBRAINS.gif"/>
 
-<Info>To opt-out of Suggested Save, go to ```Preferences > Tools > Pieces``` and click on Suggested Save.</Info>
```
Comment (jordan-pieces) [inline docs/extensions-plugins/jetbrains.mdx:162]: this is a coding thing and actually should probably be how it is laid out here - no worries ill correct this on another PR

**Review by jordan-pieces — APPROVED** [review]
nice great upgrades love moving away from "what is the point of installing" to "installing" and the other upgrades, ill make the one change i found on another PR today so no worries there, merging :)

**Pull request merged**